### DISCUSSION TITLE
Are you interested to purchase a PedalinoMini™ Case 1?

### DISCUSSION THREAD

**Discussion by alf45tar** [alf45tar/PedalinoMini#318] [Ideas]
- Kit 1 - Only the 3D printed case

- Kit 2 - 3D printed case + material (TTGO T-Display, 8 3DPT foot switches, 10 WS2812 led, 2 TRS female panel connectors, 2 MIDI panel connectors, resistors, diode, 6N137)

- Kit 2 - Fully assembled

What do you think is the correct price for each kit?

Thanks
alf45tar

**Comment by jisv48** [comment]
That will be great to have different options, here are some bunch of ideas.

1. Case — perhaps $25 to $30
2. will need to calculate the cost of:
Case  $25.00
Switches 3.00 each * 8 = $24.00
ESP32 about $15.00
Arduino Pro $10.00
TRS connectors  $2.00 = $4.00
Resistors and diodes $ $2.00
Midi connectors $2.00 = $4.00
circuit board: $3.00
LEDs .60 * 10 = $6.00
Packaging costs ???
Labor  costs  ????  I do not know, how much will cost you to put together one in hours.

If you buy in bulk, perhaps there will be quite significant less, you will need to find a supplier that will help you to put it together, maybe Sparkfun, Adafruit, etc.  How much do you want to invest and how much do you want in return, does Kickstarter maybe another option?


3. I have see in a similar device although yours it is more customizable with better support for what I have observed.



I like your product, and I will order a case immediately and the circuit board.  I have the rest in a project board.

You have a great product.

cheers,

**Reply by SnakeEaterBand** [reply]
I would be very interested in some sort of 'kit' purchase. I think it would be great for newcomers like me. I think 'kit 2' could include an instructional step by step manual/pdf download, which you could maybe create with the help of others, or while building a complete kit 3 order. Even just the purchase of a full guide could be a good option that a lot of people, myself included, would pay for. Then you have no shipping/parts/labor to worry about and could maximize your profits and sales to the community. I'm so in love with creation, but a bit daunted by the building/programming side. Like jisv48 says - you have a great product.

**Comment by troyofalltrades** [comment]
I would be interested in a complete kit or a finished product.

**Reply by troyofalltrades** [reply]
Hey any chance these will be available for purchase soon?

**Reply by alf45tar** [reply]
I don't think so

**Comment by jisv48** [comment]
Yes, how do we order it?

**Reply by alf45tar** [reply]
It is no possible yet.

**Reply by DThumbs** [reply]
Interested!

**Comment by Wapata** [comment]
Kit 2 is in my opinion the good solution. With a +5$ if the ttog is programed 

User need to be involved to handle the result, it take out the Kit 3
Not every single switch is the same, it take out the kit 1 but stl/step could be provided with kit 2.

Then if you need to run a business or gain money, do the real calculation, if you want to provide a service, take the cost, multiply by 3 or 4 and then add the postage and taxes. 

Still... it's just my opinion
Have a nice day !

**Reply by alf45tar** [reply]
thanks

**Comment by jisv48** [comment]
Yes, I’ll be interested and if you can provide the mother board that will be great or the Kit 2 as Wapata suggested. Let me know how to procede. Cheers

**Reply by alf45tar** [reply]
PedalinoMini™ Case 1 has no motherboard. Components are attached to the 3D printed case and connected by wire. Check the photos of the [inside](https://github.com/alf45tar/PedalinoMini-Case-1/blob/main/PedalinoMini-Case-1-Inside-Wired.jpg).

**Comment by jisv48** [comment]
Okay, let me know how to get the case.  Cheers
